exo-create/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/Originate/exosphere/exo-create/helpers"
+	"github.com/spf13/cobra"
+	"github.com/tmrts/boilr/pkg/template"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "exo create",
+	Short: "Create a new Exosphere application",
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) == 1 && args[0] == "help" {
+			if err := cmd.Help(); err != nil {
+				panic(err)
+			}
+			return
+		}
+		fmt.Print("We are about to create a new Exosphere application\n\n")
+		templatePath, err := helpers.CreateTemplate()
+		if err != nil {
+			log.Fatalf("Failed to create the template")
```

Comment:
Lets also output the error here so we know what the error is. Something like: `log.Fatal("Failed to create template:", err)`. Same for the other three instances in this file.